content/css/concepts/floats/floats.md

```diff
@@ -0,0 +1,73 @@
+---
+Title: 'Floats'
+Description: 'The float property moves an element to the left or right within its parent, outside of the default positioning.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Floats'
+CatalogContent:
+  - 'learn-css'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `float` property moves an element to the left or right within its containing element, outside of the default [positioning](https://www.codecademy.com/resources/docs/css/position).
+
+## Syntax
+
+The `float` property primarily uses the following three values:
+
+* The `left` value that makes the element float on the left of its container.
+* The `right` value that makes the element float on the right of its container.
+* The `none` value that causes the element not to float.
+
```

Comment:
```suggestion
* The `none` value that causes the element not to float.
```

Reply:
@Dusch4593 There's no change here. What is it supposed to be?

Reply:
Delete line  # 23 (the extra empty line)